api/rql/ast/asttest/suite.go

```diff
@@ -6,17 +6,23 @@ import (
 	"time"
 
 	"github.com/puppetlabs/wash/api/rql"
-	"github.com/puppetlabs/wash/api/rql/internal"
 	"github.com/puppetlabs/wash/api/rql/internal/errz"
 	"github.com/puppetlabs/wash/plugin"
 	"github.com/shopspring/decimal"
 
 	"github.com/stretchr/testify/suite"
 )
 
-// Suite represents a type that tests RQL AST nodes
+// Suite represents a type that tests RQL AST nodes. N represents the AST node
+// constructor that returns an empty AST node object. It can be overridden by
+// each test. DefaultN represents the default node constructor that should be
+// set when the suite class is created.
+//
+// TODO: Add some more comments once a tested version of this is working.
```

Comment:
I'm not sure what more needs to be done to address this TODO.

Reply:
Oh, yeah that's no longer needed. Thanks, updated.